bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/config/ZWaveConfiguration.java

```diff
@@ -353,6 +355,18 @@ else if(node.getSendCount() > 20 && (node.getRetryCount() * 100 / node.getSendCo
 				record.value = node.getLocation();
 				records.add(record);
 
+				ZWaveSwitchAllCommandClass switchAllCommandClass = (ZWaveSwitchAllCommandClass)node.getCommandClass(ZWaveCommandClass.CommandClass.SWITCH_ALL);
+	            if (switchAllCommandClass != null) {
+	            	record = new OpenHABConfigurationRecord(domain, "Switch All", "Switch All", false);
```

Comment:
It's preferable not to use spaces in the name. As elsewhere, I think it's better not to put this at the top level of the node.